analyzers/tests/SonarAnalyzer.UnitTest/TestCases/StringLiteralShouldNotBeDuplicated.CSharp9.cs

```diff
@@ -7,39 +7,48 @@
 
 record Record
```

Comment:
Can you please add a positional record example? As the base class does not do the containingSymbol check currently, I expect that with the current implementation we raise twice the issues.